tderenko/ithillel_mvc
# [homework-23] MVC migration and action store

# issue
Створити функціонал по міграціях.
Створити юзерс контролер та екшн store в ньому

# Test Plan
1. install project
2. run migration `docker exec ithillel_mvc-php-1 php run.php migrate`
![image](https://user-images.githubusercontent.com/18249681/189746660-6400caa7-4ec4-4957-9b7e-b7a5eae5507f.png)
3. you should see new table in database
4. go to `http://localhost:8020/user/create`
5. fill the fields and submit
6. you should see the request data
![image](https://user-images.githubusercontent.com/18249681/189747231-19887ee1-1b67-463a-ab09-6b014d4ebdd7.png)
